go.mod

```diff
@@ -3,22 +3,22 @@ module github.com/cryostatio/cryostat-operator
 go 1.18
 
 require (
-	github.com/go-logr/logr v1.2.0
-	github.com/jetstack/cert-manager v1.1.0
+	github.com/cert-manager/cert-manager v1.9.1
+	github.com/go-logr/logr v1.2.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
-	github.com/openshift/api v3.9.0+incompatible
-	k8s.io/api v0.24.0
-	k8s.io/apimachinery v0.24.0
-	k8s.io/client-go v0.24.0
+	github.com/onsi/gomega v1.19.0
+	github.com/openshift/api v0.0.0-20220712151050-2647eb31dee7
```

Comment:
Also, I wonder which source we can find this tag?

Reply:
This is from the `release-4.11` branch. I added a comment to indicate that. Go modules don't work with branches, only tags, and the OpenShift API repo doesn't tag their releases.

Reply:
Ahh that makes sense! Thanks a lot!